frontend/messages/en-US.json

```
@@ -348,6 +348,12 @@
 	"the_device_has_been_authorized": "The device has been authorized.",
 	"enter_code_displayed_in_previous_step": "Enter the code that was displayed in the previous step.",
 	"authorize": "Authorize",
+	"federated_identities": "Federated Identities",
+	"federated_identities_description": "Add external identity providers (Azure, AWS, Kubernetes, etc.) that can authenticate this client using JWT tokens instead of client secrets for enhanced security.",
```

Suggestion:

```suggestion
	"federated_identities_description": "Using federated identities, you can authenticate OAuth2 clients (apps) using JWT tokens issued by third-party authorities, including cloud platforms, Kubernetes, SPIRE, etc. When available, this is a potentially safer alternative to using client secrets.",
```

I think I can also add a docs page and link to that?

---

Looks good, maybe we can omit the examples to make the description a bit shorter? Adding a docs section would probably make sense too.

---

I think we should shorten it to something like this: 

`Using federated identities, you can authenticate OAuth2 clients using JWT tokens issued by third-party authorities. Learn More.` 

(learn more would be a link) That cleans up and doesn't add un-needed fluff/filler words.

---

I like that, I’d just maybe add “OAuth2 clients (applications)” because the terminology is confusing (unless you’re involved with the RFCs!)

---

I agree with Elias, We should keep it consistant throughout the UI.

---

Added here [`878a42f` (#566)](https://github.com/pocket-id/pocket-id/pull/566/commits/878a42f26b432240d28b2dd527f26b59caa3a2b5)